### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by bjiang27**
# Description
This PR updates the config version and image tag for csipowermax-reverseproxy to 2.11.0.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?

### CODE REVIEW

File: content/docs/authorization/v1.x/configuration/powermax/_index.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Create the karavi-authorization-config secret using this command:
         # enabled: Always set to true
         enabled: true
         forceRemoveModule: true
-        configVersion: v2.12.0
+        configVersion: v2.11.0
```
Comment (rajkumar-palani) [inline content/docs/authorization/v1.x/configuration/powermax/_index.md:145]: Both 1.x and 2.x pointing to the same config version?

File: content/docs/authorization/v2.x/configuration/powermax/_index.md [hunk 2]
```diff
@@ -116,12 +116,12 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
...
         enabled: true
         forceRemoveModule: true
-        configVersion: v2.12.0
+        configVersion: v2.11.0
         components:
         - name: csipowermax-reverseproxy
         # image: Define the container images used for the reverse proxy
         # Default value: None
-          image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.12.0
+          image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.11.0
```
Comment (rajkumar-palani) [inline content/docs/authorization/v2.x/configuration/powermax/_index.md:124]: Are we downgrading (or) wrongly updated before? Is this the latest version of reverse proxy image?

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
Can you please address the comments. Thanks.

File: content/docs/authorization/v1.x/configuration/powermax/_index.md (hunk 1 above)
Comment (bjiang27) [reply content/docs/authorization/v1.x/configuration/powermax/_index.md:145]: Last version for Reverse Proxy was v2.10.0, so it should be v2.11.0 for csipowermax-reverseproxy.

File: content/docs/authorization/v2.x/configuration/powermax/_index.md (hunk 2 above)
Comment (bjiang27) [reply content/docs/authorization/v2.x/configuration/powermax/_index.md:124]: Last version for Reverse Proxy was v2.10.0, so it should be v2.11.0 for csipowermax-reverseproxy.

**Review by sharmilarama — APPROVED** [review]
LGTM

**Review by hoppea2 — APPROVED** [review]

**Review by anandrajak1 — APPROVED** [review]

**Comment by suryagupta4** [comment]
@bjiang27: At some places, driver versions are still not updated, can you please address those in this PR? And please cross verify once if other pages are updated correctly or not, thanks.

https://github.com/dell/csm-docs/blob/main/content/docs/csidriver/_index.md?plain=1#L19
https://github.com/dell/csm-docs/blob/main/content/docs/csidriver/release/powerscale.md?plain=1#L7
https://github.com/dell/csm-docs/blob/main/content/docs/deployment/csminstallationwizard/_index.md?plain=1#L17 - changes in the table

**Review by prablr79 — APPROVED** [review]

**Pull request merged**